sqlparse/filters.py

```diff
@@ -691,7 +691,8 @@ def _create_column_definition(self, token_list):
             self._skip_white_space_and_new_line_tokens(token_queue)
         )
 
-        if col_type_token.value == 'set' or col_type_token.value == 'enum':
+        if (col_type_token.value.lower() == 'set' or
+                col_type_token.value.lower() == 'enum'):
```

Comment:
fyi: this also can be done as: `col_type_token.value.lower() in ('set', 'enum')`, which may be simpler.